src/instructlab/debug/debug.py

```diff
@@ -0,0 +1,39 @@
+# Standard
+import logging
+import os
+
+# Third Party
+from click_didyoumean import DYMGroup
+import click
+
+# First Party
+from instructlab import utils
+from instructlab.configuration import ILAB_CONFIG_HOME, ILAB_DATA_HOME
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(cls=DYMGroup)
+@click.pass_context
+def debug(ctx):
+    """Command Group for debug-related ilab calls"""
+    ctx.obj = ctx.parent.obj
+    ctx.default_map = ctx.parent.default_map
+
+
+@click.command()
+@click.pass_context
+@utils.display_params
+def reset(
```

Comment:
Two issues with debug

1. Command structure is noun-verb (this is verb verb)
2. Reseting your state is not really related to debugging

I'd go with "system" (e.g. podman system reset is similar) or @cdoern 's suggestion of "storage"

Reply:
Done!